Adjustments to docs (#3659)
spacetelescope/jdaviz: docs/imviz/plugins.rst

it to re-link by pixels or WCS as needed.

For WCS linking, the "fast approximation" option uses an affine transform

> to represent the offset between images, if possible. This method, although less accurate,
> is much more performant but should be accurate to within a pixel for most cases.
> If approximation fails, WCS linking will fall back to the full transformation.

Reviewer: ```suggestion
to represent the offset between images, if possible. This method, although less accurate,
is much more performant and should still be accurate to within a pixel for most cases.
If approximation fails, WCS linking will fall back to the full transformation.
```
I didn't like that the "but" read like it was contrasting "performant" rather than "less accurate."

    created with the :guilabel:`replace` option are acceptable as background regions
    (see :ref:`imviz_defining_spatial_regions`).

   If your image is already background subtracted, choose "Manual" and set the

>    :guilabel:`Background value` at 0.

Reviewer: ```suggestion
   :guilabel:`Background value` to 0.
```

       This is because, for performance reasons, the plugin multiplies
       by the area after the aperture sum is calculated.

7. If you also want your photometry result in the unit of counts, you can enter a
   conversion factor in the :guilabel:`Counts conversion factor` field. The value
   must be in the unit of display data unit per counts. This is used to convert linear
   flux unit (e.g., MJy/sr) to counts. This field is only used if data has a valid unit.
   If this field is not applicable for you, leave it at 0.
   **This field resets every time Data selection changes.**

> 8. If you also want photometry result in magnitude units, you can enter a flux

Reviewer: ```suggestion
8. If you also want your photometry result in magnitude units, you can enter a flux
```

    This plugin is still under active development. As a result, the search only uses the SDSS DR17 catalog and
    the Gaia catalog and works best when you only have a single image loaded in a viewer.

To load a catalog from a supported `JWST ECSV catalog file <https://jwst-pipeline.readthedocs.io/en/latest/jwst/source_catalog/main.html#output-products>`_,

> choose "From File..." from the dropdown menu.

Reviewer: ```suggestion
choose "From File..." from the menu.
```
I vaguely remember that "dropdown" is considered redundant here from the last time I was reading style guidelines.
